Zend/zend_exceptions.stub.php

```diff
@@ -22,6 +25,21 @@ public function getTraceAsString(): string;
 
 class Exception implements Throwable
 {
+    /** @var string */
+    protected $message = "";
+    /** @var string */
+    private $string = "";
+    /** @var int */
+    protected $code = 0;
+    /** @var string|null */
+    protected $file = null;
+    /** @var int|null */
+    protected $line = null;
+    /** @known */
```

Comment:
Support for `@known` isn't really needed, as the name gets interned anyway. More than anything, it's a convenience when writing the code out by hand...